app/services/events/validate_creation_service.rb

```diff
@@ -67,10 +68,10 @@ def validate_create
 
       return transaction_id_error unless valid_transaction_id?
       return invalid_code_error unless valid_code?
-      return invalid_properties_error unless valid_properties?
+      return invalid_properties_error unless valid_properties? # TODO: filters on events
 
       subscription = organization.subscriptions.find_by(external_id: params[:external_subscription_id])
-      invalid_quantified_event = quantified_event_validation(subscription || subscriptions.first)
+      invalid_quantified_event = quantified_event_validation(subscription || subscriptions.first) # TODO: quantified events valdiation
```

Comment:
Layout/LineLength: Line is too long. [134/120] (https://rubystyle.guide#max-line-length)